### ISSUE TITLE
Drop 5, custom fields and OnPropertyChanged

### ISSUE THREAD

**Issue by OlivierCC** [SharePoint/sp-dev-docs#275]
#### Category
- [ X] Question
- [ ] Typo
- [ X] Bug
- [ X] Enhancement / Suggestion
- [ ] Additional article idea

#### Observed Behavior

In the Drop 5, a part of the method OnPropertyChanged has been removed to an internal method called _internalOnPropertyChange. Now the method is called only to notify a changed, and do not update really the web part property.

So with this Drop and the new API in sp-webpart-base module, how to save a Web Part property in a custom field?

**Comment by manishgarg1** [comment]
Yes, that is a good observation Olivier. Can you please explain your requirement in a bit more detail. In the onPropertyPaneFieldChanged(propertyPath, oldValue, newValue) API should should be able to set a custom property also.  You have all the information you need. 

Are you expecting a certain call flow in this case ?

**Comment by OlivierCC** [comment]
Yes you are right & thank you for your answer. In fact in my old code, I just called the old onPropertyPaneFieldChange method to update the value of a property and refresh the WP. With the new onPropertyPaneFieldChanged method, it refreshes only the WP. But that's not a problem, I realized that I need only to update the property manually before, and now it's working:

private onCustomPropertyChanged(value: string) {
  var oldValue: string = this.properties.customProp;
  this.properties.customProp = value;
  this.onPropertyPaneFieldChanged('customProp', oldValue, value);
}
